apps/root/src/pages/strategy-guardian-detail/strategy-management/withdraw/form/index.tsx

```diff
@@ -15,11 +16,18 @@ interface WithdrawFormProps {
 
 const WithdrawForm = ({ strategy, setHeight }: WithdrawFormProps) => {
   const { withdrawAmount } = useEarnManagementState();
+  const intl = useIntl();
   return (
     <>
       <DelayedWithdrawContainer strategy={strategy} />
       <FormWalletSelector
         filter={strategy ? { chainId: strategy.network.chainId, tokens: [strategy.asset] } : undefined}
+        chipDescription={intl.formatMessage(
```

Comment:
This is not available to withdraw right? It's available in wallet, I think it's still confusing